specification/appendix/examples/metadata_examples/adding_new_columns_example.md

```diff
@@ -0,0 +1,79 @@
+# Adding New Columns
+
+## Scenario
+
+ACME has decided add additional columns to their FOCUS data export. The new columns are x_awesome_column1, x_awesome_column2, and x_awesome_column3. The provider creates a new schema object to represent the new schema, this schema object has a unique SchemaId. The subsequent data exports that use the new schema include the new schema's id as a reference to their corresponding schema object.
+
+### Supplied Metadata
+
+#### Location for the new schema object:
+
+`/FOCUS/metadata/schemas/schema-23456-abcde-23456-abcde-23456.json`
+
+#### Content for the new schema object:
+
+```json
+ {
+  "SchemaId": "23456-abcde-23456-abcde-23456",
+  "FocusVersion": "1.0",
+  "name": "New Columns",
```

Comment:
`name`... what is this?  I don't see a file that describes it.  Is it the Schema name?

Reply:
Ahhhhh, good point. This was pulled from the last one and is not part of the spec. We could pull it or have it as optional.

Reply:
This is fixed